<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hud</class>
 <widget class="QWidget" name="Hud">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="max_health_layout">
    <item>
     <widget class="QLabel" name="max_health_label">
      <property name="text">
       <string>Max Health:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="max_health_value">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="increase_max_health_btn">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="health_regen_layout">
    <item>
     <widget class="QLabel" name="health_regen_label">
      <property name="text">
       <string>Health Regen:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="health_regen_value">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="increase_health_regen_btn">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="damage_layout">
    <item>
     <widget class="QLabel" name="damage_label">
      <property name="text">
       <string>Damage:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="damage_value">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="increase_damage_btn">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="movement_speed_layout">
    <item>
     <widget class="QLabel" name="movement_speed_label">
      <property name="text">
       <string>Movement Speed:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="movement_speed_value">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="increase_movement_speed_btn">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="attack_speed_layout">
    <item>
     <widget class="QLabel" name="attack_speed_label">
      <property name="text">
       <string>Attack Speed:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="attack_speed_value">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="increase_attack_speed_btn">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="bullet_speed_layout">
    <item>
     <widget class="QLabel" name="bullet_speed_label">
      <property name="text">
       <string>Bullet Speed:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="bullet_speed_value">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="increase_bullet_speed_btn">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
